data/tribes/initialization/empire/units.lua

```diff
@@ -1300,26 +1375,42 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_charcoal_kiln",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Charcoal Kiln
+            lore = pgettext("empire_building", "‘Creta an carbone notandi?’ (Are they to be marked with chalk or charcoal?)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site:
+            lore_author = pgettext("ie. judge whether they are good or bad"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Charcoal Kiln
             purpose = pgettext("building", "Burns logs into charcoal.")
          }
       },
       {
          name = "empire_smelting_works",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Smelting Works
+            lore = pgettext("empire_building", "‘Ignis aurum probat, miseria fortes viros.’ (Fire tests gold, adversity [tests] brave men.)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site:
+            lore_author = pgettext("Saledus on the importance of experienced soldiers"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Smelting Works
             purpose = pgettext("building", "Smelts iron ore into iron and gold ore into gold.")
          }
       },
       {
          name = "empire_toolsmithy",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Toolsmithy
+            lore = pgettext("empire_building", "‘Faber est suae quisque fortunae.’ (Each man is the maker of his own fortune.)"),
```

Comment:
This one is literally "smith of his own fortune" in hungarian. "Faber" is more general though, meaning any craftsman – this is also a reasonable fit for the toolsmith making tools for everybody. This more natural English translation weakens the link too much though, so I'm not sure if we can have it…